### DISCUSSION TITLE
Prowler findings - filter by tags

### DISCUSSION THREAD

**Discussion by vadimq3** [prowler-cloud/prowler#990] [Q&A] [answered]
Hello,

 I think it would be nice if there would be a way to filter the findings which belong only to some specific group/application.

For example, I have a small application hosted on a huge AWS account. But doing the prowler assessment for my project is not very easy, as, from all the prowler findings, I have to manually filter only the resources which belong to my application.  Is there any way for the prowler findings to return the tags associated with that resource, so they can be filtered afterward?

Thanks,
Best Regards

**Comment by toniblyx** [comment, marked as answer]
Hi @vadimq3 that is an great idea and I've played few times to implement it but every service manages tags somehow different and it is a big effort to implement. We will consider it for v3 when we will make important changes in the way we run checks.
